dazzle-rewrite/dazzle.yaml

```diff
@@ -1,36 +1,58 @@
 combiner:
   combinations:
     - name: base
-    # - name: essentials
-    #   chunks:
-    #   - tool-docker
-    # - name: lang-c
-    #   ref:
-    #   - essentials
-    #   chunks:
-    #   - lang-c
-    # - name: lang-clojure
-    #   ref:
-    #   - essentials
-    #   chunks:
-    #   - lang-clojure
+      chunks:
+        - tool-docker
```

Comment:
@aledbf What other tool chunks should go in base? I am thinking `tool-tailscale` too, but nothing else.

Reply:
> What other tool chunks should go in base?

Good question. I put docker because is what makes sense.

> I am thinking tool-tailscale too, but nothing else.

Sure?

Reply:
added tailscale